lib/mixlib/install/backend/artifactory.rb

```diff
@@ -54,24 +56,31 @@ def info
           end
         end
 
-        private
+        def product_versions
+          params = {
+            g: "com.getchef",
+            a: options.product_name,
+            repos: ARTIFACTORY_REPOSITORY
+          }
+          results = client.get("/api/search/versions", params)["results"]
 
-        def artifact(result)
-          ArtifactInfo.new(
-            md5:              result["properties"]["omnibus.md5"].first,
-            sha256:           result["properties"]["omnibus.sha256"].first,
-            version:          result["properties"]["omnibus.version"].first,
-            platform:         result["properties"]["omnibus.platform"].first,
-            platform_version: result["properties"]["omnibus.platform_version"].first,
-            architecture:     result["properties"]["omnibus.architecture"].first,
-            url:              result["downloadUri"]
-          )
+          results.collect { |result| result["version"] }
+        end
+
+        def product_version
+          if options.latest_version?
+            Mixlib::Versioning.find_target_version(product_versions, nil, false, true).to_s
```

Comment:
The Opcode::Version `find_target_version` method includes prerelease and nightlies logic.  I don't think we need that.  I guess I'm not sure what the current line is missing.  What is it you are seeing as an improvement?